src/main/java/frc/robot/subsystems/swerve/factories/swerveconstants/SimulationSwerveConstants.java

```diff
@@ -7,12 +7,14 @@
 public class SimulationSwerveConstants {
 
     public static final double VELOCITY_AT_12_VOLTS_METERS_PER_SECOND = 5.054;
+    public static final String SWERVE_NAME = "Simulation";
```

Comment:
i am not really sure about all of this, butt what happens if there are two swerves, like it will be 2sim with same path, so maybe add in the factory logPath as public static that have swerve name (now will be swerve, like module have front right front left...), and insert it to the getConstants() function